experiments/20160805-mutagenesis/20160902-mutagenesis-2.md

```diff
@@ -78,3 +78,32 @@
 | 72              | 2 min           | times    |
 | 72              | 7 min           | 1        |
 | 4               | hold            |          |
+
+### Gel Electrophoresis
+
+Gel Parameters
+- 1% (w/v), 50 mL TBE
+- 150 volts, 35 min
+
+Gel Order
+
+| Ladder | P2 | PW | PT | A1 | A2 | AW | AT | Ladder |
+|--------|----|----|----|----|----|----|----|--------|
+
+![gel](./20160903-mutagenesis-2.jpg)
+
+## Analysis
+
+**Expected Sizes**
+
+| Fragment        | expected size | successful?|
+|-----------------|---------------|------------|
+| A1: PB2 Part 1  | 1600          | No...      |
+| A2: PB2 Part 2  | 820           | Faint      |
+| P2: pCI Part 2  | 1374          | YES       |
+
+- **A1**: The two brightest bands are at smaller sizes than expected, indicating fragmentation. A much fainter band between the 1.5 and 2.0 reference bands could represent DNA of the expected size.
+- **A2**: The brightest band is between the 0.5 and 1.0 kb reference bands, so DNA strand of expected size is likely. However, band is still faint.
+- **P2**: The band is between the 1.0 and 1.5 kb reference bands as expected.
+
+Since pCI part 1 was successfully produced despite having the largest size at 2.7kb, perhaps we could try PCR with the whole PB2 fragment instead of splitting it?
```

Comment:
Definitely worth a shot! I've not tried the full length fragment with the current set of primers.

For the full length protein, EM-30 should be Fw, and EM-27 should be Re. If you'd like to try it out on the weekend using the MyTaq enzyme (the Agilent enzyme is much more expensive), give it a shot! (But no pressure to do so.)

We can chat on Tuesday - I'll show you a bit more of the software I wrote to compute cloning primers and store them in the database.